hw/ip/alert_handler/data/alert_handler.hjson

```diff
@@ -184,53 +185,103 @@
                 '''
       swaccess: "rw",
       hwaccess: "hro",
-      regwen:   "REGWEN",
+      regwen:   "PING_TIMER_REGWEN",
       fields: [
         {
-          # TODO: add PING_CNT_DW parameter here
-          bits: "23:0",
+          bits: "PING_CNT_DW-1:0",
           resval:   32,
-          desc: '''Timeout value in cycles. If an alert receiver or an escalation sender does not
+          desc: '''
+          Timeout value in cycles. If an alert receiver or an escalation sender does not
           respond to a ping within this timeout window, a pingfail alert will be raised.
           '''
         }
       ]
     }
+    { name:     "PING_TIMER_EN",
+      desc:     '''
+                Ping timer enable.
+                '''
+      swaccess: "rw1s",
+      hwaccess: "hro",
+      regwen:   "PING_TIMER_REGWEN",
+      fields: [
+        {
+          bits: "0",
+          resval:   0,
+          desc: '''
+          Setting this to 1 enables the ping timer mechanism.
+          This bit cannot be cleared to 0 once it has been set to 1.
+
+          Note that the alert pinging mechanism will only ping alerts that have been enabled and locked.
+          '''
+        }
+      ]
+    }
 # all alerts
-    {skipto: "0x20"},
+    { multireg: { name:     "ALERT_REGWEN",
+                  desc:     "Register write enable for alert enable bits.",
+                  count:    "NAlerts",
+                  compact:  "false",
+                  swaccess: "rw0c",
+                  hwaccess: "hro",
+                  hwqe:     "false",
+                  cname:    "alert",
+                  fields: [
+                    { bits:   "0",
+                      name:   "EN",
+                      desc:   '''
+                              Alert configuration write enable bit.
+                              If this is cleared to 0, the corresponding !!ALERT_EN
+                              and !!ALERT_CLASS bits are not writable anymore.
+
+                              Note that the alert pinging mechanism will only ping alerts that have been enabled and locked.
+                              ''',
+                      resval: "1",
+                    }
+                  ]
+                }
+    },
     { multireg: { name:     "ALERT_EN",
                   desc:     '''Enable register for alerts.
                   ''',
                   count:    "NAlerts",
+                  compact:  "false",
                   swaccess: "rw",
                   hwaccess: "hro",
-                  regwen:   "REGWEN",
+                  regwen:   "ALERT_REGWEN",
+                  regwen_multi: "true",
```

Comment:
we really need a better name for this field...do you have any suggestions?
i know i came up with `regwen_multi`, but i feel like it's very non-intuitive. Is it just me?

Reply:
`bitwise_regwen`?

Reply:
i think that's better... maybe we should take a vote hehe.